pkg/sfu/streamtracker.go

```diff
@@ -121,6 +121,21 @@ func (s *StreamTracker) Stop() {
 	}
 }
 
+func (s *StreamTracker) Reset() {
+	s.Stop()
+
+	s.countSinceLast = 0
+	s.cycleCount = 0
+
+	s.initMu.Lock()
+	s.initialized = false
```

Comment:
re-using the init logic during reset will trigger multiple `detectWorker` to start. seems that we don't want that.

Reply:
`Reset` calls `Stop` first which closes `s.running` channel which will end the current `detectWorker`. Can you please explain how multiple `detectWorker`s will start?

Reply:
maybe I'm overthinking this.. Stop() doesn't immediately stop the detector, which checks `IsRunning` periodically in a loop.. As long as we have a reasonable amount of time between Reset and receiving packets again (like how it's currently being used), it wouldn't be an issue.

when we move Reset to immediately before resume, then it feels like the timing issue is more prominent? perhaps it's still much longer than the detect loop.

Reply:
Got it. No overthinking. Thank you for catching these edge cases. These are the ones which will come back to bite at the most inopportune time :-). I will think about how to address this.